### DISCUSSION TITLE
Getting the title from the 2nd level

### DISCUSSION THREAD

**Discussion by ottz0** [statamic/cms#7651] [General]
I want to get the page title from the 2nd level of my sites collection. Can't find this in the docs.

`{{title:depth:2}}`

Otherwise I can only see segment_2 as a url so I need to unhypen and capitalise

**Comment by jasonvarga** [comment]
```antlers
{{ get_content from="/{segment_1}/{segment_2}" }}
    {{ title }}
{{ /get_content }}
```
